app/src/main/java/org/xwiki/android/sync/Constants.kt

```diff
@@ -50,6 +50,12 @@ const val PAGE_SIZE = 30
 
 const val XWIKI_DEFAULT_SERVER_ADDRESS = "https://www.xwiki.org/xwiki"
 
+const val ACCESS_TOKEN = "access_token"
+
+const val TEST_USERNAME = "aa700"
```

Comment:
Please, move test things to the test sources

Reply:
Make a constant file for test and put it there ?

Reply:
Move these constants to androidTests sources